docs/basic-features/data-fetching.md

```diff
@@ -220,6 +221,16 @@ function Blog({ posts }: InferGetStaticPropsType<typeof getStaticProps>) {
 export default Blog
 ```
 
+Note: by default Next.js has a timeout for static generation where if no activity occurs for 60 seconds it will re-attempt generation three times and if it still fails after retrying the build will fail. This timeout can be increased if needed via the below config:
```

Comment:
```suggestion
Note: Next.js has a default timeout for static generation where if no activity occurs for 60 seconds it will re-attempt generation three times. If the fourth attempt fails, the build will fail. This timeout can be increased if needed using the below configuration:
```